libraries/AP_RangeFinder/AP_RangeFinder_Benewake_TFMiniPlus.cpp

```diff
@@ -69,7 +69,7 @@ AP_RangeFinder_Backend *AP_RangeFinder_Benewake_TFMiniPlus::detect(
 bool AP_RangeFinder_Benewake_TFMiniPlus::init()
 {
     const uint8_t CMD_FW_VERSION[] =         { 0x5A, 0x04, 0x01, 0x5F };
-    const uint8_t CMD_SYSTEM_RESET[] =       { 0x5A, 0x04, 0x04, 0x62 };
+    const uint8_t CMD_SYSTEM_RESET[] =       { 0x5A, 0x04, 0x02, 0x60 };
```

Comment:
i'd suggest we remove CMD_SYSTEM_RESET and remove this line:
```
_dev->transfer(CMD_SYSTEM_RESET, sizeof(CMD_SYSTEM_RESET), nullptr, 0);
```
as we only call this on startup, and the sensor should already be reset

Reply:
need to read datasheet and check if any of our settings only take effect after reset

Reply:
also check if 250Hz is actually a good idea

Reply:
> also check if 250Hz is actually a good idea

https://github.com/ArduPilot/ardupilot/pull/16102#issue-771335366



> need to read datasheet and check if any of our settings only take effect after reset

[15936180690810637SJ-PM-TFmini Plus A05 Product Manual.pdf](https://github.com/ArduPilot/ardupilot/files/10025300/15936180690810637SJ-PM-TFmini.Plus.A05.Product.Manual.pdf)